chemprop/nn/utils.py

```diff
@@ -48,7 +48,7 @@ def get_activation_function(activation: str | Activation) -> nn.Module:
 
 class OutputTransform:
```

Comment:
can both of these be implemented as `nn.Module`s? That is, they should expect tensors as input and produce tensors as outputs. We should not be using `numpy` internally in these classes. I'm thinking something like:

```python
class OutputTransform(nn.Module):
    mean: Tensor
    std: Tensor

    def __init__(self, means: ArrayLike, stds: ArrayLike):
        super().__init__()

        self.register_buffer("mean", torch.tensor(means))
        self.register_buffer("std", torch.tensor(stds))

    def forward(self, Y: Tensor):
        return Y if self.training else self.mean + self.std * Y 
```

It might help to define a class method for convenience:

```python
@classmethod
def from_standard_scaler(cls, scaler: StandardScaler) -> OutputTransform:
    return cls(scaler.mean_, scaler.scale_)
```